README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/fedosovdn/lesson_4_homework.svg?branch=%D0%B2%D1%8B%D0%BF%D0%BE%D0%BB%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5_%D0%94%D0%97_%E2%84%964)](https://travis-ci.org/2gisprojectT/lesson_4_homework)
```

Comment:
[!(https://travis-ci.org/2gisprojectT/lesson_4_homework.svg?branch=%D0%B2%D1%8B%D0%BF%D0%BE%D0%BB%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5_%D0%94%D0%97_%E2%84%964)](https://travis-ci.org/2gisprojectT/lesson_4_homework)